disconnected/mirroring/about-installing-oc-mirror-v2.adoc

```diff
@@ -118,3 +106,14 @@ include::modules/oc-mirror-imageset-config-parameters-v2.adoc[leveloffset=+1]
 // Command reference for oc-mirror v2
 include::modules/oc-mirror-command-reference-v2.adoc[leveloffset=+1]
 * xref:../../disconnected/mirroring/about-installing-oc-mirror-v2.adoc#oc-mirror-updating-cluster-manifests-v2_about-installing-oc-mirror-v2[Configuring your cluster to use the resources generated by oc-mirror]
+
+[id="next-steps_{context}"]
+== Next steps
+
+After you mirror images to your disconnected environment using oc-mirror plugin v2, you can perform any of the following actions:
+
+* xref:../../disconnected/installing.adoc#installing-disconnected-environments[Installing a cluster in a disconnected environment]
+* xref:../../disconnected/using-olm.adoc#olm-restricted-networks[Using Operator Lifecycle Manager in disconnected environments]
+* xref:../../disconnected/updating/disconnected-update-osus.adoc#updating-disconnected-cluster-osus[Updating a cluster in a disconnected environment using the OpenShift Update Service]
```

Comment:
question: does this doc tell the customer that the osus graph image is what oc-mirror generated already? and that they don' t need to build an image?

Reply:
I believe so. In [this section](https://docs.openshift.com/container-platform/4.17/disconnected/updating/disconnected-update-osus.html#update-service-graph-data_updating-disconnected-cluster-osus) for creating the graph data image, there is a note before the procedure telling users that oc-mirror creates the image and that oc-mirror users can skip the procedure.

If that's not what you are asking about then please let me know